sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Field.java

```diff
@@ -759,4 +770,35 @@ public Field setFields(List<Field> fields) {
         this.fields = fields;
         return this;
     }
+
+    /**
+     * Get the hidden property: A value indicating whether the field will be
+     * returned in a search result. This property must be true for key fields,
```

Comment:
You'll want to fix the documentation to be negated, I have a feeling the `key` field shouldn't be hidden.

```suggestion
     * returned in a search result. This property must be false for key fields,
```

Reply:
Most of the javadoc is the same as .net. Will double check